src/TodoListItem.jsx

```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types'
+
+const TodoListItem = (props) => {
+    return(
+        <li>{props.todo.title}</li>
+    )
+}
+
+TodoListItem.propTypes = {
+    todo: PropTypes.shape({

```

Comment:
if you'd like, this can even be refactored into a reusable type, so you don't have to re-define both in this file and in the file above